WIP global rephrasing (#222)
meilisearch/documentation: guides/introduction/quick_start_guide.md

  </a>
</p>

> Since all the source code from the GitHub repository will be compiled, the deployment process can take up to 20 minutes to complete.

Reviewer: I think the whole heroku part can be removed.

Author: Removed!

::::

> [Environment variables and flags](/guides/advanced_guides/installation.md#environment-variables-and-flags) can be set before and on launch. Among other things, you can use them to add the **master key** or set the **port**.

Reviewer: ```suggestion
[Environment variables and flags](/guides/advanced_guides/installation.md#environment-variables-and-flags) can be set before and on launch. Amongst all the flags, there is the **master key** and the **port** flags that you can use.
```

Author: Maybe:

```suggestion
[Environment variables and flags](/guides/advanced_guides/installation.md#environment-variables-and-flags) can be set before and on launch. Amongst all the flags, you can use the **master key** and the **port** flags.
```

### Checking updates

> Most actions are asynchronous, which allows you to stack them. All of the actions are executed in the order in which they were specified.

Reviewer: ```suggestion
Most actions are asynchronous, which allows you to stack them. All of the actions are executed in the order in which they were sent.
```
